.lighthouse/lighthouseCommentMaker.js

```diff
@@ -12,7 +12,6 @@ ${scoreRow('Performance', summary.performance)}
 ${scoreRow('Accessibility', summary.accessibility)}
 ${scoreRow('Best practices', summary['best-practices'])}
 ${scoreRow('SEO', summary.seo)}
```

Comment:
The `next-pwa` plugin is abandoned and required a lot of tweaking to make work with the `app` folder. Wasn't worth the effort, so I just removed it and suppressed the LH test results.